internal/db/sqltest/Makefile

```diff
@@ -32,6 +33,7 @@ test:
 	@echo Using $(PG_TAP_DOCKER_IMAGE)
 	@docker run -d \
 		--name $(SQL_TEST_CONTAINER_NAME) \
+		-p $(TEST_DB_PORT):5432 \
```

Comment:
Can you make this consistent with the `SQL_TEST_CONTAINER_NAME` env var, so make this `SQL_TEST_DB_PORT`

Reply:
done